src/worker.rs

```diff
@@ -18,8 +18,13 @@ use crate::metrics;
 use crate::settings::Policy;
 use crate::utils::convert_yaml_map_to_json;
 
+struct PolicyEvaluatorWithSettings {
+    policy_evaluator: PolicyEvaluator,
+    allowed_to_mutate: Option<bool>,
+}
```

Comment:
I would change the `PolicyEvaluator` struct to have the optional `allowed_to_mutate` field. In this way, if we will ever want to offer this function also to kwctl, we will have all the things we need.

Plus, I personally prefer to have this attribute into `PolicyEvaluator`, rather then create a wrapper to hold it.

Reply:
I thought we agreed on not going in that direction (https://github.com/kubewarden/policy-evaluator/pull/71).

If you folks agree that this should go in the policy-evaluator go ahead. I don't think it's the case, given I believe this is a behavior of what to do with the result of the policy-evaluator.

Reply:
Yeah, scratch it. I'm fine with the current implementation of this PR